en/components/grids_templates/column_selection.md

```diff
@@ -0,0 +1,166 @@
+@@if (igxName === 'IgxGrid') {
+---
+title: Angular Grid Column Selection | Ignite UI for Angular | infragistics
+_description: Learn how to configure column selection with the Angular Material table. The column selection makes the interaction with the Grid more appealing than ever.
+_keywords: column selection, igniteui for angular, infragistics
+---
+}
+@@if (igxName === 'IgxTreeGrid') {
+---
+title: Angular Tree Grid Column Selection | Ignite UI for Angular | infragistics
+_description: Learn how to configure column selection with the Angular Material table. The column selection makes the interaction with the Tree Grid more appealing than ever.
+_keywords: column selection, igniteui for angular, infragistics
+---
+}
+@@if (igxName === 'IgxHierarchicalGrid') {
+---
+title: Angular Hierarchical Grid Column Selection | Ignite UI for Angular | infragistics
+_description: Learn how to configure column selection with the Angular Material table. The column selection makes the interaction with the Hierarchical Grid more appealing than ever.
+_keywords: column selection, igniteui for angular, infragistics
+---
+}
+
+### @@igComponent Column Selection
+The **column selection feature** provides an easy way to select an entire column with a one click action. It **emphasizes** the importance of a particular column by focusing on the header cell(s) and everything below. The feature comes with a rich [`API`]({environment:angularApiUrl}) that allows manipulation of the **state**, **data extraction** of the **selected** fractions easing **data analysis** operations and visualizations.
```

Comment:
allows manipulation of the selection **state**, **data extraction** from the **selected** fractions and **data analysis** operations and visualizations.